modules/contact-form/grunion-contact-form.php

```diff
@@ -2725,6 +2741,8 @@ function process_submission() {
 			$spam = '***SPAM*** ';
 		}
 
+		$in_comment_disallowed_list = apply_filters( 'jetpack_contact_form_in_comment_disallowed_list', false, $akismet_values );
```

Comment:
```suggestion
		/**
		 * Filter whether a submitted contact form is in the comment disallowed list.
		 *
		 * @module contact-form
		 *
		 * @since 8.9.0
		 *
		 * @param bool  $result         Is the submitted feedback in the disallowed list.
		 * @param array $akismet_values Feedack values returned by the Akismet plugin.
		 */
		$in_comment_disallowed_list = apply_filters( 'jetpack_contact_form_in_comment_disallowed_list', false, $akismet_values );
```

Reply:
Thank you for fixing this!